### ISSUE TITLE
showHorizontalScopeLine: false disables vertical lines too

### ISSUE THREAD

**Issue by aNickzz** [CoenraadS/BracketPair#148]
With the following config

```json
{
	"bracketPairColorizer.showVerticalScopeLine": true,
	"bracketPairColorizer.showHorizontalScopeLine": false
}
```

**Expected**
Colored vertical scoping lines (per example image in readme).

**Actual**
Only the default whitish vertical lines are shown.

Setting `editor.renderIndentGuides: false` results in no lines being shown.

**Comment by CoenraadS** [comment]
Can I ask if you want horizontal lines fully disabled, or only disabled when the scope is one line  
 e.g. `(blah blah)`

**Comment by aNickzz** [comment]
Fully disabled, I find it overlaps the squiggly red lines sometimes making them harder to see/notice.

```php
    if ($this->misspelledFunctione() === null) {
        //blah
    }
```

**Comment by CoenraadS** [comment]
Should be fixed in v1.0.56
